config.yaml.sample

```diff
@@ -21,10 +21,24 @@ tiles:
       url: https://raw.githubusercontent.com/mapzen/metroextractor-cities/master/cities.json
       zoom-start: 11
       zoom-until: 15
+
+      # Can be set to an array of metro-extract city names, like
+      # 'new-york_new-york', to only seed tiles from those areas rather than
+      # all of them.
+      cities: Null
```

Comment:
Can we make this something that is valid yaml? Either: `cities:`, `cities: null`, or `cities: [new-york_new-york]`.